example/nodejs/teardown.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function () {
+
+    var path = require('path');
+    var provider = require(path.resolve(__dirname, 'provider.js'));
+
+    try {
+        provider.verify(); // Verify that all the mock calls has been made, throw error if not
```

Comment:
In fact, `mockService` is what we're calling it everywhere. Here too would be good.